genienlp/ned/scripts/oracle_vs_bootleg.py

```diff
@@ -0,0 +1,98 @@
+import argparse
```

Comment:
Should this and `analyze_bootleg_results` be added as genienlp scripts?

Reply:
These scripts are for local data analysis. I don't want to crowd genienlp commands with data processing/ analysis scripts.